.github/workflows/build-git-installers.yml

```diff
@@ -333,13 +335,33 @@ jobs:
 
       - name: Build payload
         run: |
+          die () {
+            echo "$*" >&2
+            exit 1
+          }
+
           # Configure the environment
-          set -x
+          set -ex
           PATH=/usr/local/bin:$PATH
           export CURL_LDFLAGS=$(curl-config --libs)
 
           # Write to "version" file to force match with trigger payload version
           echo "${{ needs.prereqs.outputs.tag_version }}" >>git/version
+
+          # Configure the Git build to pick up gettext
+          homebrew_prefix="$(brew --prefix)"
+          cat >git/config.mak <<EOF
+          LDFLAGS = -L$homebrew_prefix/lib -L/usr/local/opt/gettext/lib
+          CFLAGS = -I$homebrew_prefix/include -I/usr/local/opt/gettext/include
+          EOF
+
+          # On Apple Silicon, homebrew apparently does not install a `gcc` symlink
+          test x86_64 = '${{ matrix.arch.name }}' ||
+          echo 'CC = gcc-13' >>config.mak
```

Comment:
I cannot confirm this. I don't have homebrew's version of gcc/clang installed. I have Xcodetools (which does a bunch of magic of its own).  I'm not saying we should switch, just that I can't confirm this.

Reply:
i have installed homebrew GCC on my M1 and this is present.